### ISSUE TITLE
Download exits (seemingly before complete)

### ISSUE THREAD

**Issue by Rob-T** [skyme5/magzdb#10]
* magzdb version: 1.1.1
* Python version: 3.7.3
* Operating System: Debian 10.5

### Description
Magzdb quits downloading and exits with the following,

```
Traceback (most recent call last):
  File "/home/user01/.local/bin/magzdb", line 10, in <module>
    sys.exit(main())
  File "/home/user01/.local/lib/python3.7/site-packages/magzdb/cli.py", line 87, in main
    filter=args.filter,
  File "/home/user01/.local/lib/python3.7/site-packages/magzdb/magzdb.py", line 214, in download
    r'''<a\s*href\=\.\.\/file\/(?P<id>\d+)/dl>''',
AttributeError: 'NoneType' object has no attribute 'group'
```

### What I Did
I've had this error several times, but in this instance I ran the following,
`/home/user01/.local/bin/magzdb -i 1654 -f "year >= 2009"`
```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```

**Comment by skyme5** [comment]
Probably because Magzdb.com doesn't provide download link for pdf. How frequent is this?

**Comment by Rob-T** [comment]
Yes, that seems to be the reason - no download link. For example: magzdb -i 1654 -e 3827048

Perhaps instead of quitting, it could carry on with the download task (for example, a range of editions) and then report 'some files not downloaded' and report the missing links in a log?

**Comment by skyme5** [comment]
This behaviour is already implemented in [release v1.1.2](https://github.com/skyme5/magzdb/releases/tag/v1.1.2)

**Comment by Rob-T** [comment]
Excellent!
